docs/enterprise-app-store/index.md

```diff
@@ -5,6 +5,8 @@ tags: [enterprise app store, in-house distribution, enterprise distribution, faq
 ---
 
 import ContentRef from '@site/src/components/ContentRef';
+import PATDanger from '@site/docs/testing-distribution/\_orgAtoB-send-binary-danger.mdx';
```

Comment:
The more correct and conventional form is `PatDanger`.
Using PascalCase, where each word starts with a capital letter, commonly used for naming classes or components in many programming languages.